functions/split.yml

```diff
@@ -0,0 +1,100 @@
+name: split
+topic: strings
+args:
+  required:
+  - name: subject
+    type: [string]
+    desc: 'Subject string'
+  - name: search
+    type: [string]
+    desc: 'Text separator'
+  optional:
+  - name: count
+    type: [number]
+    desc: 'Number of splits to perform'
+returns:
+  type: ['array[string]']
+  desc: ''
+desc: |
+  Given the `subject` string, `split()` breaks on ocurrences of the string `search` and returns an array.
+
+  The `split()` function returns an array containing each partial string between occurrences of `search`.
+  If `subject` contains no occurrences of the `search` text, an array containing just the original `subject`
```

Comment:
It sure does in my current implementation.

Reply:
But it needs to be specified, lest implementations disagree. And which rule wins for `split('', '')`—is that `[]` or `['']`?

Reply:
Hum 🤔 that is a good catch.

It could even return `` [ "", "" ] `` 😲 But this seems to contradict the original premises.
Let me rethink about the whole use case here…

Reply:
#118